website/source/docs/guides/containers-guide.md

```diff
@@ -0,0 +1,328 @@
+* `server_stabilization_time` should be sufficiently large (on the order of several seconds) so that unstable servers are not added to the cluster until they stabilize.
+
+If the container running the currently-elected Consul server leader is stopped, a leader election will be triggered. This event will cause a new Consul server in the cluster to assume leadership.
+
+When a previously stopped server container is restarted using `docker start <container_id>`,  and it is configured to obtain a new IP, autopilot will add it back to the set of Raft peers with the same node-id and the new IP address, after which it can participate as a server again.
+
+
+### Backing-up Data
+
+You can back-up your Consul cluster using the [consul snapshot](https://www.consul.io/docs/commands/snapshot.html) command. 
+
+```sh
+$ docker exec <container_id> consul snapshot save backup.snap
+```
+
+This will leave the `backup.snap` snapshot file inside of your container. If you are not saving your snapshot to a [persistent volume](https://docs.docker.com/storage/volumes/) then you will need to use `docker cp` to move your snapshot to a location outside of your container.
+
+```sh
+$ docker cp <container_id>:backup.snap ./ 
+```
+
+Users running the Consul Enterprise Docker containers can run the [consul snapshot agent](https://www.consul.io/docs/commands/snapshot/agent.html) to save backups automatically. Consul Enterprise's snapshot agent also allows you to save snapshots to Amazon S3 and Azure Blob Storage.
+
+### Environment Variables
+
+You can also add configuration by passing the configuration JSON via environment variable `CONSUL_LOCAL_CONFIG`. 
+
+```sh
+$ docker run \
+	-d \
+	-e CONSUL_LOCAL_CONFIG='{
+	"datacenter":"us_west",
+	"server":true,
+	"enable_debug":true
+	}' \
+	consul agent -server -bootstrap-expect=3
+```
+
+Setting `CONSUL_CLIENT_INTERFACE` or `CONSUL_BIND_INTERFACE` on `docker run` is equivalent to passing in the `-client` flag(documented [here](https://www.consul.io/docs/agent/options.html#_client)) or `-bind` flag(documented [here](https://www.consul.io/docs/agent/options.html#_bind)) to Consul on startup.
+
+Setting the `CONSUL_ALLOW_PRIVILEGED_PORTS` runs setcap on the Consul binary, allowing it to bind to privileged ports. Note that not all Docker storage backends support this feature (notably AUFS). 
+
+```sh
+$ docker run -d --net=host -e 'CONSUL_ALLOW_PRIVILEGED_PORTS=' consul -dns-port=53 -recursor=8.8.8.8
+```
+
+## Summary
+
+In this guide you learned to deploy a containerized Consul cluster. You also learned how to deploy a containerized service and how to configure your Consul client to register that service with your Consul cluster.
+
+Further steps can be taken to deploy and secure a Consul cluster that is production ready, connect to other clusters in other datacenters, or deploy additional microservices that can find each other with Consul service discovery and connect securely with Consul Connect.
```

Comment:
This sentence is really long and kind of hard to read, could be broken up.

Reply:
Updated!